src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -1357,7 +1362,7 @@ public String takeScreenshot(String basename) {
         return screenshotFile;
     }
 
-    private String getScreenshotLink(String screenshotFile) {
+    public String getScreenshotLink(String screenshotFile) {
```

Comment:
Making this protected would suffice, right? This is not something to be called from the wiki.